iCount/genomes/__init__.py

```diff
@@ -59,7 +59,7 @@ def species(source='gencode', release=None):
     if source == 'gencode':
         return gencode.species()
     else:
-        return ensembl.species(release=release)
+        return ensembl.species(release=gencode._to_int(release))  # pylint:disable=protected-access
```

Comment:
Why call a `gencode`'s function when preparing the arguments for an `ensembl` call?
Move function `_to_int()` into genomes/__init__.py and make it a "general function" or have a separate function that handles release strings for `encode`.

Reply:
Good point. I moved functions `_to_int` and `get_ftp_instance` to `__init__.py`